L1Trigger/TrackFindingTracklet/src/TrackletLUT.cc

```diff
@@ -235,33 +235,43 @@ void TrackletLUT::initmatchcut(unsigned int layerdisk, MatchType type, unsigned
   name_ = settings_.combined() ? "MP_" : "MC_";
 
   if (type == barrelphi) {
+    nbits_ = 10;
```

Comment:
Where do the numerical values, such as "10" that nbits_ is assigned to come from? Can they be deduced from https://twiki.cern.ch/twiki/bin/view/CMS/HybridDataFormat ? Are these numbers of bits not already stored in the Settings.h file, so we can could take them from there?

Reply:
This PR failed git CI, because "scram b -j code-format" was not run before making it.